src/Executor/Executor.php

```diff
@@ -83,8 +85,8 @@ public function createRuntime(
             'variables' => $variables,
             'remove' => $remove,
             'command' => $command,
-            'cpus' => $this->cpus,
-            'memory' => $this->memory,
```

Comment:
We can remove those $this variables, they shouldn' be needed anymore

Reply:
Alright, done

Reply:
I meant those:

![CleanShot 2024-07-10 at 09 16 16@2x](https://github.com/appwrite/appwrite/assets/19310830/1bfc8fc0-c4bd-4cf0-869b-b0826b2af7e1)

Reply:
Whoops, my bad. Fixing now